Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Themes/InteractableCustomSetting.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.UI
+{
+    /// <summary>
```

Comment:
What is this for?

Reply:
I updated the comments. It's used by the themes to expose properties for additional theme settings. Allows more customization of themes.

Reply:
Used in the GrabScaleTheme